src/main/resources/application.conf

```diff
@@ -39,4 +39,4 @@ templatecontrol {
 include "templates"
 include "2.5.x"
 include "2.6.x"
-
+include "2.7.x"
```

Comment:
We should eventually rename this `.conf` files to `play-2.7.x.conf` to avoid confusion with lagom ones. (separate PR)

Reply:
that will be done when we have support for Lagom